sdk/communication/communication-chat/src/models/models.ts

```diff
@@ -2,14 +2,16 @@
 // Licensed under the MIT license.
 
 import { CommunicationIdentifier, CommunicationIdentifierKind } from "@azure/communication-common";
-import { ChatError, ChatMessageType } from "../generated/src";
+import { ChatAttachment, ChatError, ChatMessageType } from "../generated/src";
 
 export {
   AddChatParticipantsResult,
   ChatMessageType,
   ChatThreadItem,
   ChatError,
   SendChatMessageResult,
+  ChatAttachment,
+  AttachmentType,
```

Comment:
I directly export `ChatAttachment` and `AttachmentType` from generated models to the public. Avoid some duplicated models copy